build.js

```diff
@@ -82,7 +82,7 @@ const fencedCodeBlockDefinition = (name, identifiers, sourceScope, language, add
 
 	return `fenced_code_block_${name}:
   begin:
-    (^|\\G)(\\s*)(\`{3,}|~{3,})\\s*(?i:(${identifiers.join('|')})((\\s+|:|\\{)[^\`~]*)?$)
+    (^|\\G)(\\s*)(\`{3,}|~{3,})\\s*(?i:(${identifiers.join('|')})((\\s+|:|\\{?)[^\`~]*)?$)
```

Comment:
I think you need to escape the `?`: `\?`. The current fixtures show the JS/python highlighting isn't kicking in properly

Reply:
You know, I thought the same thing and on my machine it broke if I escaped it with `\\?`. Let me double check that I wasn't just misreading the output. :-)

Reply:
Ah, yes, I see I just didn't understand how the test suite was working. Fixed now!